docs/src/main/asciidoc/firestore.adoc

```diff
@@ -255,6 +255,29 @@ Now when you call the methods annotated with `@Transactional` on your service ob
 If an error occurs during the execution of a method annotated with `@Transactional`, the transaction will be rolled back. 
 If no error occurs, the transaction will be committed.
 
+==== Subcollections
```

Comment:
Can you rename your pull request to "Add subcollection support for FirestoreTemplate" to be more specific?